nuxeo-coldstorage-web/src/main/resources/web/nuxeo.war/ui/coldstorage/nuxeo-coldstorage.html

```diff
@@ -57,3 +60,27 @@
     <nuxeo-coldstorage-infobar document="[[document]]"></nuxeo-coldstorage-infobar>
   </template>
 </nuxeo-slot-content>
+
+<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" disabled> </nuxeo-slot-content>
+
+<nuxeo-slot-content name="coldStorageReplaceBlobAction" slot="BLOB_ACTIONS">
+  <template>
+    <nuxeo-coldstorage-replace-blob-action document="[[document]]"></nuxeo-coldstorage-replace-blob-action>
```

Comment:
As I think about the approach to fork the element, perhaps we could use a `nuxeo-filter` here instead and this way we would not have to maintain the fork. Something like this:

```
<nuxeo-slot-content name="replaceBlobAction" slot="BLOB_ACTIONS" order="35" priority="1">
  <template>
    <nuxeo-filter
      document="[[document]]"
      expression="!this.hasFacet(document, 'ColdStorage')"
    >
      <template>
        <nuxeo-replace-blob-button document="[[document]]" xpath="[[xpath]]"></nuxeo-replace-blob-button>
      </template>
    </nuxeo-filter>
  </template>
</nuxeo-slot-content>
```
If the above works, I think we could even use the same for the other two actions!